jenkins/pipeline-parallel-mtr.groovy

```diff
@@ -238,7 +238,9 @@ void setupTestSuitesSplit() {
             wget \${RAW_VERSION_LINK}/${BRANCH}/mysql-test/mysql-test-run.pl -O ${WORKSPACE}/mysql-test-run.pl
             chmod +x ${WORKSPACE}/suites-groups.sh
             set +e
-            ${WORKSPACE}/suites-groups.sh check ${WORKSPACE}/mysql-test-run.pl
+            # Set WORKER_x_MTR_SUITES and check if suites list is consistent with the one specified in mysql-test-run.pl
+            source ${WORKSPACE}/suites-groups.sh
```

Comment:
Yes, we source it to get split variables, but I'm not sure if it is the right way to call the function directly from sourced script. In my opinion more readable is calling a script as it was done before

Reply:
We source only functions now, not variables. 
https://stackoverflow.com/questions/10822790/can-i-call-a-function-of-a-shell-script-from-another-shell-script